Problems/1018/ans_1018_JHP.py

```diff
@@ -0,0 +1,37 @@
+# https://www.acmicpc.net/problem/1018
+# 문제 : 8x8짜리 체스판으로 잘라낸 후에 칠해야하는 최소의 정사각형 갯수
+
+import sys
+
+def solve():
+    input = sys.stdin.readline
+    N, M = map(int,input().split())
+    board = []
+    result = []
+    
+    for _ in range(N):
+        board.append(input())
+
+    # 브루트포스 풀기
+    for i in range(N-7): # 8x8로 자르기 위해 i-7, j-7만큼 고정 필요 (끝에서 7번째)
+        for j in range(M-7): # 상동 이유
+            first_W = 0
+            first_B = 0
+            for k in range(i,i+8): # k~k+8까지 색칠하는 횟수 카운트
+                for v in range(j,j+8):
+                    if (k+v)%2 == 0: # 한칸씩 띄어가며 확인
+                        if board[k][v] != 'W':
+                            first_W += 1
+                        else:
+                            first_B += 1
+                    else:
+                        if board[k][v] != 'B':
+                            first_W += 1
+                        else:
+                            first_B += 1
+        result.append(first_B)
+        result.append(first_W)
```

Comment:
```suggestion
            result.append(first_B)
            result.append(first_W)
```

파이썬에서 다중 반복문을 걸면 머리아파지는 이유...
indent-rainbow extension을 씁시다.

Reply:
Rainbow extension을 사용했는데...append가 다중 루프를 돌리다보니 i에 대해 update하는 걸로 잘못 생각하고 짰더라고요....로직 자체는 맞는 접근이었으나 휴먼 에러였습니다....

Reply:
Rainbow extension이 어떤걸 말하는거에요?

Reply:
> Rainbow extension이 어떤걸 말하는거에요?

저저번 세미나 때 소개 했던 VS Code extension 중 하나